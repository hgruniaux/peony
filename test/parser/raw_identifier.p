//# RUN: peony %s

fn foo() {}

fn r#i32() {
    let r#i32 = 5;
    if r#i32 == 8 {}
    r#foo();

    r#f();
    //~^ ERROR: use of undeclared identifier 'f'
}
