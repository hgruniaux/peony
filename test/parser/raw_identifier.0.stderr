<source>:10:5: error: use of undeclared identifier 'f'
   10 |     r#f();
      |     ~~~
